packages/itwinui-react/src/core/InputGroup/InputGroup.tsx

```diff
@@ -121,18 +125,29 @@ export const InputGroup = React.forwardRef((props, forwardedRef) => {
       >
         {children}
       </Box>
-      {(message || status || svgIcon) && (
-        <StatusMessage
-          iconProps={{
-            'aria-hidden': true,
-          }}
-          startIcon={svgIcon}
-          status={status}
-          {...messageProps}
-        >
-          {displayStyle !== 'inline' && message}
-        </StatusMessage>
-      )}
+      {(() => {
```

Comment:
Probably would be better to move this into a different component (in the same file), both for readability and performance

Reply:
Sounds good, done (https://github.com/iTwin/iTwinUI/pull/1877/commits/d5e0d4173d348e04631a22ebe6cdbcfa7eeef3bf).

Reply:
the curlies `{}` can also go.